src/index.html

```diff
@@ -1,6 +1,18 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
```

Comment:
Remember to use 2 spaces for indentation in your file. The indentation here seems to be more than 2 spaces.